pkg/kapp/config/default.go

```diff
@@ -582,6 +594,16 @@ changeRuleBindings:
       - notMatcher:
           matcher: *disableDefaultChangeGroupAnnMatcher
 
+- rules:
+  - "upsert before upserting change-groups.kapp.k14s.io/packageinstall"
+  - "upsert before upserting change-groups.kapp.k14s.io/appcr"
+  - "delete after deleting change-groups.kapp.k14s.io/packageinstall"
+  - "delete after deleting change-groups.kapp.k14s.io/appcr"
+  ignoreIfCyclical: true
+  resourceMatchers:
+  - anyMatcher: {matchers: *serviceAccountMatchers}
+  - anyMatcher: {matchers: *rbacMatchers}
```

Comment:
we should apply disableDefaultChangeGroupAnnMatcher logic similar to above so that users in emergency case can opt out of ordering.

Reply:
Updated